script/build/mk

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+set -e
+
+pkg_topdir=$(cd $(dirname $0)/../../ && pwd -P)
+
+[ "${pkg_os}" = "" ] && pkg_os=`uname -s | tr -s 'A-Z' 'a-z'`
+[ "${pkg_arch}" = "" ] && pkg_arch=`uname -m`
+
+pkg_name=measurement-kit
+
+pkg_dist_dir=${pkg_topdir}/MK_DIST/${pkg_os}/${pkg_arch}/${pkg_name}
+rm -rf ${pkg_dist_dir}
+mkdir -p ${pkg_dist_dir}
+
+pkg_build_dir=${pkg_topdir}/MK_BUILD/${pkg_os}/${pkg_arch}/${pkg_name}
+rm -rf ${pkg_build_dir}
+mkdir -p ${pkg_build_dir}
+
+(
+    set -e
+
+    ./autogen.sh # Must run in the toplevel directory
+
+    cd ${pkg_build_dir}
+
+    if [ "$pkg_os" = "windows" ]; then
+        # XXX Not implemented
+	exit 1
+    fi
+
+    ../../../../configure --prefix=${pkg_dist_dir} --disable-shared           \
+        --disable-examples                                                    \
+        --disable-binaries                                                    \
+        --with-geoip=${pkg_dist_dir}/../geoip                                 \
+        --with-libevent=${pkg_dist_dir}/../libevent                           \
+        --with-openssl=${pkg_dist_dir}/../libressl                            \
+        ${pkg_configure_flags}
+
+    make V=0 -j4 install
```

Comment:
Maybe the parallelism should be something configurable?

Reply:
I'd say, it should be either hardcoded or configurable with default [`MAKEFLAGS`](https://github.com/darkk/home/blob/master/.bashrc#L61-L78). Making additional configuration option to `make` sounds like unnecessary complexity to me.